.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Deprecations? | yes/no
+| Tests pass?   | yes/no
+
+---
+
+## Before
+
+< The current status >
+
+## After
+
+< What will be changed >
+
+## Screenshots
+
+< If required. [LICEcap](http://www.cockos.com/licecap/) for Mac or SilentCast for Linux allow you to great GIFs from screen capture >
+
+## Comments
+
+< Anything you would like the reviewer to note >
```

Comment:
So in the end, the tpl could be split like this

---

_(label notes_)

## Overview

## Before

## After

## Tests

## Technical details

## Additional comments